src/programQueueProcessor/MockFetch.ts

```diff
@@ -48,7 +50,7 @@ export const mockAnalysisFetcher: typeof fetchAnalyses = async ({
   studyId,
   rdpcUrl,
   analysisType,
-  isMutect,
+  workflowName,
```

Comment:
Was going to suggest using the enum type for this prop, but this argument has no types so... bigger problems we need to deal with.